client/visualizer/webpack.config.js

```diff
@@ -10,11 +10,12 @@ var conf = {
   context: path.resolve(__dirname, 'src'),
   entry: {
     app: './app.ts',
+    profiler: './profiler.ts'
   },
   output: {
     path: path.resolve(__dirname, 'bc20'),
     publicPath: '/bc20/',
-    filename: 'bundle.js'
+    filename: '[name].js'
```

Comment:
ohhhh i see

Reply:
Sidebar has been made bigger to make space for the Profiler tab. As a consequence the gamearea canvas has been made smaller.